ui/app/templates/components/console/log-json.hbs

```diff
@@ -8,7 +8,8 @@
     @showToolbar={{false}}
     @value={{stringify this.content}}
     @readOnly={{true}}
-    @viewportMargin="Infinity"
+    {{! ideally we calculate the "height" of the json data, but 100 should cover most cases }}
```

Comment:
previously this was defaulting to 10, so another option is to remove this all together and let it default to 10 until someone complains?

Reply:
From what I can gather this is output for the ui console/terminal window. 10 for most situations should be enough unless they're trying to read a really long secret. 

Is 100 going to have any performance implications? not that I can tell via testing, but also there's no documentation to say otherwise. From what I gather, they're cautioning against using "Infinity" because that could be bad if your json was millions of lines long.